src/mds/PurgeQueue.cc

```diff
@@ -65,7 +65,6 @@ void PurgeItem::decode(bufferlist::const_iterator &p)
       decode(stamp, p);
       decode(pad_size, p);
       p.advance(pad_size);
-      decode((uint8_t&)action, p);
```

Comment:
this looks incorrect. don't you've to still decode action as done in the parent commit?

Reply:
@ajarr Thanks for pointing that out! Fixed it.